.github/issue-labeler.yml

```diff
@@ -0,0 +1,90 @@
+# archived/webui:
+# backport/1.6:
+# backport/1.7:
+# backport/1.8:
+# beta/1.8:
+# blocks-release:
+# crash:
+# do-not-merge:
+# help-wanted:
+# needs-discussion:
+# needs-investigation:
+# post-beta:
+# pr/dependencies:
+# pr/needs-rebase:
+# pr/needs-tests:
+theme/acls:
+    - '(acl|ACL)'
+# theme/agent-cache:
+# theme/api:
+theme/catalog:
+    - '(catalog|Catalog)'
+theme/certificates:
+    - '(certificate|CA|Certificate\sAuthority)'
+theme/cli:
+    - '(command-line|commandline|cli|command)'
+theme/config:
+    - '(configuration_files|options#configuration_files)'
+theme/connect:
+    - '(Connect|connect|service\smesh|mesh|proxy|gateway)'
+theme/consul-nomad:
+    - '(nomad|Nomad)'
+theme/consul-vault:
+    - '(vault|Vault)'
+# theme/contributing:
+theme/dns:
+    - '(dns|DNS)'
+theme/envoy/xds:
+    - '(envoy|xds|xDS|XDS|Envoy)'
+theme/federation-usability:
+    - '(WAN|wan|wan_fed|federation|fed)'
+# theme/go-modules:
+theme/health-checks:
+    - '(health-check|healthcheck|health\scheck)'
+theme/ingress-gw:
+    - '(ingress|ingress-gateway|Ingress)'
+# thementernal-cleanup:
+theme/internals:
+    - '(raft|RAFT|SERF|serf|lifeguard|Lifeguard|gossip)'
+theme/kubernetes:
+    - '(kubernetes|k8s|helm)'
+theme/mesh-gw:
+    - 'mesh'
+# theme/operator-usability:
+theme/packaging:
+    - '(rpm|deb|repository|systemd|upstart|Upstart|SystemD)'

```

Comment:
Do we also want to capture things such as brew, choco, etc?

Reply:
That's a great idea! That said, I asked product-delivery, and this was the list the gave me so far. I'll add brew, choco, and nix.